### PR TITLE
Update Ticket Raising Form

### PR DESCRIPTION
**Pull request by ssian2**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/DTSPO-6719
https://tools.hmcts.net/jira/browse/DTSPO-7294

### Change description ###

This PR:

- Adds a 'Team' Field to the ticket raising form that will add a tag to the raised JIRA ticket.
- Removes the 'Action Required' field from the form as it is not being used.
- Adds an optioal documentation form when closing a ticket

**Does this PR introduce a breaking change?** (check one with "x")

```
[x] Yes
[ ] No
```

### CODE REVIEW

File: src/messages.js [hunk 1]
```diff
@@ -562,36 +575,23 @@ function superBotMessageBlocks(inputs) {
...
+                "initial_value": inputs?.team_check?.value ?? ""
             }
         },
         {
             "type": "input",
             "block_id": 'user_block',
             "label": {
                 "type": "plain_text",
-                "text": "Ticker Raiser"
+                "text": "Ticket Raiser"
```
Comment (timja) [inline src/messages.js:594]: why do we have this?

Comment (ssian2) [reply src/messages.js:594]: We need to provide a user to the workflow so it knows who to put under the 'Reporter' field in the actual ticket. The old method just gets this from the body of the event, but I don't think you can do that with a workflow, hence why we're passing it in here

Comment (timja) [reply src/messages.js:594]: needs checking.

Comment (timja) [reply src/messages.js:594]: (discussed on slack, this works but is overly complex and the form should be moved completely to code, with pre-request bits staying in the workflow)

**Pull request merged**